libs/react-helpers/src/lib/grid-cells/flash-cell.tsx

```diff
@@ -8,84 +8,84 @@
  * @author John <john.walley@mulberryhousesoftware.com>
  */
 
-import * as React from 'react'
+import * as React from 'react';
 import { theme } from '@vegaprotocol/tailwindcss-config';
 import { splitAt } from '../format';
-  
-const FLASH_DURATION = 800 // Duration of flash animation in milliseconds
+
+const FLASH_DURATION = 800; // Duration of flash animation in milliseconds
 
 export interface FlashCellProps {
   /**
-  * The string representation of value. It can be formatted in bespoke ways,
-  * so we can't simply convert value
-  */
-  children: string
+   * The string representation of value. It can be formatted in bespoke ways,
+   * so we can't simply convert value
+   */
+  children: string;
   /** The numeric representation of 'children' */
-  value: number
+  value: number;
 }
 
 /**
-* Given two strings, finds the index first character that has changed in string 2
-*
-* Used by <FlashCell /> to highlight substrings to highlight the portion of a number that has changed
-* From: https://stackoverflow.com/questions/32858626/detect-position-of-first-difference-in-2-strings
-*
-* @param a String to diff
-* @param b Second string to diff
-* @return number Index of first different character, or -1
-*/
+ * Given two strings, finds the index first character that has changed in string 2
+ *
+ * Used by <FlashCell /> to highlight substrings to highlight the portion of a number that has changed
+ * From: https://stackoverflow.com/questions/32858626/detect-position-of-first-difference-in-2-strings
+ *
+ * @param a String to diff
+ * @param b Second string to diff
+ * @return number Index of first different character, or -1
+ */
 export function findFirstDiffPos(a: string, b: string): number {
-  let i = 0
-  if (a === b) return -1
-  if (!a || !b) return -1
+  let i = 0;
+  if (a === b) return -1;
+  if (!a || !b) return -1;
 
-  while (a[i] === b[i]) i++
-  return i
+  while (a[i] === b[i]) i++;
+  return i;
 }
 
 /**
-* Get value from previous render
-*/
+ * Get value from previous render
+ */
 function usePrevious<T>(value: T): T | undefined {
-  const ref = React.useRef<T | undefined>()
+  const ref = React.useRef<T | undefined>();
```

Comment:
chore (non-clocking): as a follow up I think we should enable full project linting on CI, noticing a lot of files getting in with little linting errors and not sure how

Reply:
Agree. There is precommit hook but it's possible that some code changes will sneak to master unnoticed